detox/test/e2e/utils/snapshot.js

```diff
@@ -0,0 +1,58 @@
+const fs = require('fs-extra');
+const { ssim } = require('ssim.js');
```

Comment:
I've added this util for image comparison

Reply:
I would advise eventually to use https://github.com/americanexpress/jest-image-snapshot instead of low-level wrappers.